<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFileSystemWidget</class>
 <widget class="QWidget" name="AddFileSystemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addFilesButton">
       <property name="text">
        <string>Add Files...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addDirectoryButton">
       <property name="text">
        <string>Add Directory...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchDirectoryButton">
       <property name="text">
        <string>Search Directory...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="loadDataCheckBox">
        <property name="text">
         <string>Load data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showDataExtentsCheckBox">
        <property name="text">
         <string>Show data extents</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeSelectedFilesButton">
     <property name="text">
      <string>Remove Selected Files</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
